src/UI/Components/ComposerCanvas/Helpers/connectEntities.ts

```diff
@@ -0,0 +1,31 @@
+import { dia } from "@inmanta/rappid";
+import { Link, ServiceEntityBlock } from "../Shapes";
+
+/**
+ * Function that create connection/link between two Entities
+ *
+ * @param {dia.Graph} graph JointJS graph object
+ * @param {ServiceEntityBlock} source JointJS shape object
+ * @param {ServiceEntityBlock} target JointJS shape object
+ * @param {boolean} isBlocked parameter determining whether we are showing tools for linkView
+ * @returns {void}
+ */
```

Comment:
`targets` should be a list, no?
```suggestion
/**
 * Function that create connection/link between two Entities
 *
 * @param {dia.Graph} graph JointJS graph object
 * @param {ServiceEntityBlock} source JointJS shape object
 * @param {ServiceEntityBlock[]} target JointJS shape objects
 * @param {boolean} isBlocked parameter determining whether we are showing tools for linkView
 * @returns {void}
 */
```

Reply:
Updated!